awsutil/awsutil.go

```diff
@@ -173,3 +178,7 @@ func NewSession(meta ECSTaskMeta, userAgentCaller string) (*session.Session, err
 	}
 	return clientSession, nil
 }
+
+func GetAWSRegion() string {
+	return os.Getenv(AWSRegionEnvVar)
```

Comment:
I think taskARN has region can we use that?

Reply:
Region is embedded in the taskARN which brings in additional challenges where we have to parse and extract it out. `AWS_REGION` is something that ECS injects automatically into the container.